xarray/backends/api.py

```diff
@@ -281,6 +281,14 @@ def maybe_decode_store(store, lock=False):
             engine = _get_default_engine(filename_or_obj,
                                          allow_remote=True)
         if engine == 'netcdf4':
+            import netCDF4
+            if len( filename_or_obj ) == 88 and float(netCDF4.__version__[:3]) < 1.3:
```

Comment:
Use `LooseVersion()` here: https://stackoverflow.com/questions/11887762/compare-version-strings-in-python